src/components/TupleSelectWithRegexConditional/TupleSelectWithRegexConditional.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { action } from "@storybook/addon-actions";
+import TupleSelectWithRegexConditional from ".";
+
+export default {
+  title: "Components/TupleSelect/TupleSelectWithRegexConditional",
+  component: TupleSelectWithRegexConditional,
+};
+
+const options = [
+  {
+    value: "build_variant",
+    displayName: "Build Variant",
+    placeHolderText: "Search Build Variant names",
+  },
+  {
+    value: "task",
+    displayName: "Task",
+    placeHolderText: "Search Task names",
+  },
+];
+
+export const Default = () => (
```

Comment:
I think we don't want the nested folder:

<img width="285" alt="Screen Shot 2023-01-27 at 12 15 56 PM" src="https://user-images.githubusercontent.com/47064971/215152732-d9679558-7273-4bec-8e6d-c2cf089d2e71.png">

Changing the path/name will get rid of the nested folder:

```suggestion
export default {
  title: "Components/TupleSelect",
  component: TupleSelectWithRegexConditional,
};

const options = [
  {
    value: "build_variant",
    displayName: "Build Variant",
    placeHolderText: "Search Build Variant names",
  },
  {
    value: "task",
    displayName: "Task",
    placeHolderText: "Search Task names",
  },
];

export const TupleSelectWithConditional = () => (
```

<img width="286" alt="Screen Shot 2023-01-27 at 12 21 51 PM" src="https://user-images.githubusercontent.com/47064971/215152791-db39005a-e6d8-46cb-a2c1-0292100a1721.png">